tests/integration/targets/mso_user/tasks/main.yml

```diff
@@ -28,6 +28,18 @@
     user: ansible_test2
   register: cm_remove_user
```

Comment:
remove the register on all 3 in the clean environment part. You could combine all 3 with a loop.